### DISCUSSION TITLE
skip checkup passes, but building project fails with Gradle run error

### DISCUSSION THREAD

**Discussion by Wheels00** [skiptools/skip#74] [Q&A]
skip checkup --double-check passes just fine (screenshot below)

I have downloaded the SkipWeather example, and I am trying to run it in both simulators. Build fails with this error:

![Screenshot 2023-12-28 at 5 18 37 pm](https://github.com/skiptools/skip/assets/12003601/f544cc0a-ad8a-4fac-b888-10e3269f5fa2)

Any suggestions?

[Update: Same error on new projects]

![Screenshot 2023-12-28 at 5 28 24 pm](https://github.com/skiptools/skip/assets/12003601/52f744dc-bbf5-4713-87c7-aed53f65fc97)

**Comment by X901** [comment]
I have same error on every skip example project 
Datebake, Showcase, SkipWeather

But when create new project it working fine
also this project run without any issue
https://github.com/DominatorVbN/SkipTour

**Reply by Wheels00** [reply]
Thanks @X901 . So I just tested that and, yes, I can run that SkipTour project just fine. 

However , a new project built with skip init has the same error.

Very frustrating.

I guess I will convert this to an issue, given that it’s possible to get this error in a new project yet still pass skip checkup

**Reply by marcprux** [reply]
Thanks for the report. I see it was also opened as an issue at https://github.com/skiptools/skip/issues/77, so I've followed-up there with some more diagnostic information you might help us with, as well as a potential workaround.

Thanks for reporting the issue, and thanks for your patience during the tech preview.
